pkg/eval/builtin_fn_io.go

```diff
@@ -787,9 +789,10 @@ func fopen(name string) (vals.File, error) {
 //
 // @cf fopen
 
-func fclose(f vals.File) error {
-	return f.Close()
-}
+// Deprecated, written in file module instead.
+// func fclose(f vals.File) error {
```

Comment:
Do not comment out a unit test for a deprecated feature. The tests should continue to run until the deprecated feature is actually removed.